types/configs/sanitizer-config.d.ts

```diff
@@ -31,5 +31,5 @@ export interface SanitizerConfig {
    *   }
    * }
    */
-  [key: string]: boolean|{[attr: string]: boolean|string}|(() => any);
+  [key: string]: boolean|{[attr: string]: boolean|string}|((el?: any) => any);
```

Comment:
why do you use `any` type instead of `Element`?

Reply:
Reasonable